File: spring-batch-plus-sample/spring-batch-plus-clear-run-id-incrementer/src/test/java/com/navecorp/spring/batch/plus/sample/RunSamplesTest.java
package com.navecorp.spring.batch.plus.sample;

import org.junit.jupiter.api.Test;

public class RunSamplesTest {

	@Test
	void all() throws Exception {
		com.navercorp.spring.batch.plus.sample.clear.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.clearwithid.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.comparison.bad.BatchApplication.main(new String[0]);
		com.navercorp.spring.batch.plus.sample.comparison.good.BatchApplication.main(new String[0]);
	}
}

File: spring-batch-plus-sample/spring-batch-plus-clear-run-id-incrementer-kotlin/src/test/kotlin/com/navercorp/spring/batch/plus/sample/RunSamplesTest.kt
package com.navercorp.spring.batch.plus.sample

import org.junit.jupiter.api.Test

class RunSamplesTest {

    @Test
    fun all() {
        com.navercorp.spring.batch.plus.sample.clear.main()
        com.navercorp.spring.batch.plus.sample.clearwithid.main()
    }
}
